tests/stockfish/test_models.py

```diff
@@ -195,12 +192,27 @@ def test_set_elo_rating(self, stockfish):
             "d2d4",
             "c2c4",
             "f1e2",
+            "c2c3",
+            "f1b5",
+            "g2g3",
+            "h2h3",
         )
         assert stockfish.get_parameters()["UCI_Elo"] == 1350
 
+        stockfish.set_elo_rating(2850)
+        major_version = stockfish.get_stockfish_major_version()
+
+        expected_best_moves = ["d2d4", "b1c3", "c2c3", "c2c4", "f1b5", "f1e2"]
+        if major_version >= 12 and not (stockfish.is_stockfish_major_version_a_date()):
```

Comment:
why there is checking for ```not (stockfish.is_stockfish_major_version_a_date()```?

Reply:
If the major version is >= 12, the engine is likely SF 12, 13, or 14, but it's possible it could be an old development build. If this is the case, f1e2 might end up being a move that's played.